src/modals/vm-export/VmExportDataProvider.js

```diff
@@ -40,10 +40,16 @@ async function validateThinVm (vmId, sdId) {
   const vm = await engineGet(`api/vms/${vmId}`)
 
   // If a VM has both attributes set it's thin/dependent
-  if (vm.original_template.id === emptyGuid || vm.template.id === emptyGuid) {
+  if (vm.template.id === emptyGuid) {
```

Comment:
```suggestion
  if (vm.original_template?.id === emptyGuid || vm.template?.id === emptyGuid) {
```
You are basically checking if vm.original_template is not null before accessing id? Then this should do the trick. Nice thing in JS :)